lua/entities/gmod_wire_expression2/core/selfaware.lua

```diff
@@ -203,6 +203,34 @@ e2function number entity:canSetName(string name)
 	return IsValid(this) and isOwner(self, this) and canSetName(self, this, name) and 1 or 0
 end
 
+--[[******************************************************************************]]--
+-- Extensions
+
+[nodiscard]
+e2function array getExtensions()
+	return table.Copy(E2Lib.GetExtensions())
+end
+
+local getExtensionStatus = E2Lib.GetExtensionStatus
+[nodiscard]
+e2function table getExtensionStatus()
```

Comment:
Can you raise the op cost to like 30+?

Reply:
I could probably just pre-generate this since extensions can't change without E2 reload. That would be better, right?

Reply:
Yeah would be much better. Was just worried about people spamming this per tick